BetterHolidays/markets/cache.py

```
@@ -1,42 +1,68 @@
 
     def pop(self, key, default=NOT_SET):
-        if default == NOT_SET:     
-            return self.cache.pop(key)
+        try:
+            if default == NOT_SET:
+                try:
+                    item = Day.select().where(Day.market == self.market, Day.date == key).get()
+                    if item:
+                        item.delete_instance()
+                        return item
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)